app/src/main/java/com/example/mytrainingpal/model/view_models/PreferencesViewModel.kt

```diff
@@ -10,6 +10,7 @@ import com.example.mytrainingpal.prefrences.PreferencesConstants.DEFAULT_BREAK_K
 import com.example.mytrainingpal.prefrences.PreferencesConstants.NOTIFICATION_DAYS_KEY
 import com.example.mytrainingpal.prefrences.PreferencesConstants.NOTIFICATION_TIME_KEY
 import com.example.mytrainingpal.prefrences.PreferencesConstants.USERNAME_KEY
+import com.example.mytrainingpal.prefrences.PreferencesDataStoreHelper
```

Comment:
There is a spelling mistake here: `prefrences` `-> Preferences`. an e is missing

Reply:
-> please rename the package so that the spelling mistake is gone :)

Reply:
done ^.^